constants.js

```diff
@@ -87,6 +87,17 @@ const constants = {
         'restore': true,
         'torrent': true,
     },
+
+    // Headers supported by AWS that we do not currently support.
+    encryptionHeaders: [
```

Comment:
I would generalize to unsupportedHeaders.  I would do an object for both as well.  Or as discussed an array if we loop through this and then check the reqHeaders.